### ISSUE TITLE
"Quiet" mode

### ISSUE THREAD

**Issue by devinhurley-ge** [alangpierce/sucrase#240]
It'd be nice to have a "quiet" mode like `-q` in the babel cli where the print statements during sucrase' compilation are not displayed in the terminal.

**Comment by alangpierce** [comment]
Makes sense. Note that on linux and macOS, you could do `sucrase (args) > /dev/null` for something similar, I think.

Out of curiosity, what's your build setup like? For example, are you compiling for node or the browser, and are you using a bundler or other tools? The use cases I've seen have typically involved running Sucrase using other tools (webpack, gulp, etc.) rather than from the CLI.

**Comment by alangpierce** [comment]
Note that Babel 7 changed the CLI args to hide the output files by default and print a summary at the end, and got rid of the `--quiet` mode. For now I'll just implement `--quiet`, but in the future I may release a breaking change to switch to something like the new way that Babel does it.
